db/sql/migrations/updates/20190111111557_add_protocolVersion_column_to_peers.sql

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Note that the migrations folder changed since https://github.com/LiskHQ/lisk/pull/2745

Please, move this file to `storage/sql/migrations/updates` and make sure your migration name has a date greater than the last one in this folder.

Reply:
Done, thanks :)